test/projection_gradients.jl

```diff
@@ -216,4 +216,57 @@ end
         end
         @test all(case_p .> 0) && all(case_d .> 0)
     end
+
+    @testset "Power Cone" begin
+        function det_case_pow_cone(x, α; dual=false)
+            v = dual ? -x : x
+            s = MOI.PowerCone(α)
+            if MOD._in_pow_cone(v, s)
+                return 1
+            elseif MOD._in_pow_cone(-v, MOI.dual_set(s))
+                return 2
+            elseif abs(v[3]) <= 1e-8
+                return 3
+            else
+                return 4
+            end
+        end
+
+
```

Comment:
what is the goal of deterministically fixing multiple seeds? We can either fix one and pull random numbers, or not fix it before the loop, and set the seed to a random number that we fix

Reply:
I think I totally misinterpreted a conversation we had awhile ago. I changed this in the tested to be like what's in the other tests.